pkg/actions/karpenter/create.go

```diff
@@ -0,0 +1,7 @@
+package karpenter
+
+// Create creates a Karpenter installer task and waits for it to finish.
+func (i *Installer) Create() error {
+	taskTree := NewTasksToInstallKarpenter(i.Config, i.StackManager, i.CTL.Provider.EC2(), i.KarpenterInstaller)
+	return doTasks(taskTree)
+}
```

Comment:
could the tasks live in this file? I know we use the `tasks.go` conventioin in `pkg/action/addons` but thats because your either using tasks or the func directly. Here we only ever use the tasks, so I feel the two can go together. wdyt?

Reply:
I'm thinking about when we add the update feature. But I'll have a thought about that.

Reply:
oh fair point. idm either way then